FluentUI/src/main/java/com/microsoft/fluentui/contextualcommandbar/CommandItem.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.fluentui.contextualcommandbar
+
+import android.support.annotation.DrawableRes
+import android.view.View
+
+interface CommandItem {
+
+    interface OnItemClickListener {
+        fun onItemClick(item: CommandItem, view: View)
+    }
+
+    /**
+     * The identifier of this item.
+     */
+    fun getId(): String? = null
```

Comment:
in general context : id is an integer as consumer define it in 
ids.xml file 
can you create 2 variants - one getId - > Int?  and another  getTag ->String ?

Reply:
Thanks, that would make sense and should work for all purposes